### PR TITLE
Create vinita.json

### PR DESCRIPTION
**Pull request by VinitaMenaria**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6491"></a>

### CODE REVIEW

File: vinita.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  "name": "Vinita Menaria",
+  "type": "personal",
+  "bio": "open source",
+  
+  "links" :  [
+  {  "name" : "follow me on insta",
+      "url" : ""https://instagram.com/vinita_menaria_?igshid=ZDdkNTZiNTM="
+      "icon" : "FaInstagram"
+   }
+ ]
```
Comment (SaraJaoude) [inline vinita.json:11]:
```suggestion
 "url": "https://instagram.com/vinita_menaria_?igshid=ZDdkNTZiNTM=",
 		"icon": "FaInstagram"
 	}]
 }
```

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please move your file to the `data` folder and make sure that your file name matches your GitHub username exactly. Instead of `vinita.json` it should be `data/VinitaMenaria.json`

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

**Comment by harshit-paneri** [comment]
> ## Fixes Issue
> ## Changes proposed
> ## Check List (Check all the applicable boxes)
> * [x]  My code follows the code style of this project.
> * [x]  My change requires changes to the documentation.
> * [x]  I have updated the documentation accordingly.
> * [x]  All new and existing tests passed.
> * [x]  This PR does not contain plagiarized content.
> * [x]  The title of my pull request is a short description of the requested changes.
> 
> ## Screenshots
> ## Note to reviewers
> [](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6491)

Hey @VinitaMenaria , your vinita.json file is not in the data folder, move this file to data folder otherise PR will not accept.

**Comment by harshit-paneri** [comment]
> > ## Fixes Issue
> > ## Changes proposed
> > ## Check List (Check all the applicable boxes)
> > 
> > * [x]   My code follows the code style of this project.
> > * [ ]  My change requires changes to the documentation.
> > * [ ]  I have updated the documentation accordingly.
> > * [x]   All new and existing tests passed.
> > * [x]   This PR does not contain plagiarized content.
> > * [x]   The title of my pull request is a short description of the requested changes.
> > 
> > ## Screenshots
> > ## Note to reviewers
> > [](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6491)
> 
> Hey @VinitaMenaria , your vinita.json file is not in the data folder, move this file to data folder otherwise PR will not accept.

**Comment by SaraJaoude** [comment]
> Hey @VinitaMenaria , your vinita.json file is not in the data folder, move this file to data folder otherwise PR will not accept.

@harshit-paneri as mentioned to you previously, please check the history of the PR and avoid duplicating suggestions already made.

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the following Docs which will help with the suggested changes the maintainers have proposed:

Quickstart: https://linkfree.io/docs/quickstart

Editing your Profile: https://linkfree.io/docs/how-to-guides/editing

**Pull request closed without merging**